test/check-application

```diff
@@ -2305,6 +2305,13 @@ class TestApplication(testlib.MachineCase):
         b.set_input_text("#run-image-dialog-volume-0-container-path-group input", "/somepath")
         validateField("#run-image-dialog-volume-0-container-path-group", "", "not be empty", resetValue="/somepath")
 
+        # Test validation of environment variables
+        b.click('.env-form .btn-add')
+        b.set_input_text("#run-image-dialog-env-0-key-group input", "sometext")
+        b.set_input_text("#run-image-dialog-env-0-value-group input", "sometext")
+        validateField("#run-image-dialog-env-0-key-group", "", "must not be empty", resetValue="sometext")
+        validateField("#run-image-dialog-env-0-value-group", "", "must not be empty", resetValue="sometext")
```

Comment:
Why actually? Unix wise it is completely legit, and sometimes even useful to define an env variable with an empty value.